crates/bevy_render/src/texture/image_loader.rs

```diff
@@ -98,6 +105,20 @@ impl AssetLoader for ImageLoader {
             let mut bytes = Vec::new();
             reader.read_to_end(&mut bytes).await?;
             let image_type = match settings.format {
+                ImageFormatSetting::FromContent => {
+                    let image_crate_format =
+                        image::guess_format(&bytes).map_err(|err| FileTextureError {
+                            error: TextureError::ImageError(err),
+                            path: format!("{}", load_context.path().display()),
```

Comment:
```suggestion
                            path: load_context.path().display().to_string(),
```

`to_string` can be used here.

Reply:
thanks, i also modified the one i copypasted from